test/api/unit/services/S3PublishedFileLister.test.js

```diff
@@ -1,143 +1,146 @@
 
         callback(null, {
           Contents: [
             { Key: `${prefix}/abc`, Size: 123 },
             { Key: `${prefix}/abc/def`, Size: 456 },
             { Key: `${prefix}/ghi`, Size: 789 },
-          ]
-        })
-      }
+          ],
+        });
+      };
 
-      factory.site({ demoBranch: "demo-branch-name" }).then(model => {
-        site = model
-        return S3PublishedFileLister.listPublishedFilesForBranch(site, "demo-branch-name")
-      }).then(publishedFiles => {
+      factory.site({ demoBranch: 'demo-branch-name' }).then((model) => {
+        site = model;
+        return S3PublishedFileLister.listPublishedFilesForBranch(site, 'demo-branch-name');
+      }).then((publishedFiles) => {
         expect(publishedFiles).to.deep.equal([
-          { name: "abc", size: 123 },
-          { name: "abc/def", size: 456 },
-          { name: "ghi", size: 789 },
-        ])
-        done()
-      }).catch(done)
-    })
+          { name: 'abc', size: 123 },
+          { name: 'abc/def', size: 456 },
+          { name: 'ghi', size: 789 },
+        ]);
+        done();
+      }).catch(done);
+    });
 
-    it("should resolve with a list of files for a preview branch", done => {
-      let site
+    it('should resolve with a list of files for a preview branch', (done) => {
+      let site;
 
       AWSMocks.mocks.S3.listObjects = (params, callback) => {
-        const prefix = `preview/${site.owner}/${site.repository}/preview`
-        expect(params.Bucket).to.equal(config.s3.bucket)
-        expect(params.Prefix).to.equal(prefix)
+        const prefix = `preview/${site.owner}/${site.repository}/preview`;
+        expect(params.Bucket).to.equal(config.s3.bucket);
+        expect(params.Prefix).to.equal(prefix);
 
         callback(null, {
           Contents: [
             { Key: `${prefix}/abc`, Size: 123 },
             { Key: `${prefix}/abc/def`, Size: 456 },
             { Key: `${prefix}/ghi`, Size: 789 },
-          ]
-        })
-      }
+          ],
+        });
+      };
 
-      factory.site({ defaultBranch: "master" }).then(model => {
-        site = model
-        return S3PublishedFileLister.listPublishedFilesForBranch(site, "preview")
-      }).then(publishedFiles => {
+      factory.site({ defaultBranch: 'master' }).then((model) => {
+        site = model;
+        return S3PublishedFileLister.listPublishedFilesForBranch(site, 'preview');
+      }).then((publishedFiles) => {
         expect(publishedFiles).to.deep.equal([
-          { name: "abc", size: 123 },
-          { name: "abc/def", size: 456 },
-          { name: "ghi", size: 789 },
-        ])
-        done()
-      }).catch(done)
-    })
-
-    it("should reject with an error if S3.listObjects is unsuccessful", done => {
-      AWSMocks.mocks.S3.listObjects = (params, cb) => cb(new Error("Test error"))
-
-      factory.site().then(model => {
-        return S3PublishedFileLister.listPublishedFilesForBranch(site, "preview")
```

Comment:
My suspicion would be that maybe somewhere `site = "asdf"` exists without a `var`, `let`, or `const` declaration 😳.